src/Common/Core/Impl/Shell/IFileDialog.cs

```diff
@@ -29,5 +29,25 @@ public interface IFileDialog {
         /// <param name="title"></param>
         /// <returns>Full path to the file selected, or <c>null</c>.</returns>
         string ShowSaveFileDialog(string filter, string initialPath = null, string title = null);
+
+        /// <summary>
+        /// Show the export image dialog box.
+        /// </summary>
+        /// <param name="imageArguments"></param>
+        /// <param name="filter"></param>
+        /// <param name="initialPath"></param>
+        /// <param name="title"></param>
+        /// <returns></returns>
+        ExportImageParameters ShowExportImageDialog(ExportArguments imageArguments, string filter, string initialPath = null, string title = null);
```

Comment:
`ShowExportImageDialog` and `ShowExportPdfDialog` aren't file-system related dialogs, so they don't belong to this interface. Again, they are specific to their features.